[helm-oci] Add docker-registry secrets to OCI doc (#813)
fluxcd/source-controller: docs/spec/v1beta2/helmrepositories.md

  password: 123456
```

> For OCI Helm repositories, Kubernetes secrets of type `docker-registry` are also supported.

Reviewer: Please add here an example:

```sh
flux create secret oci ghcr-auth \
  --url=ghcr.io \
  --username=flux \
  --password=${GITHUB_PAT}
```

> For OCI Helm repositories, it is also possible to use a kubernetes secret of type `docker-registry`.

Reviewer 2: maybe?

```suggestion
For OCI Helm repositories, Kubernetes' secrets of type `docker-registry` is also supported.
```

  password: 123456
```

> For OCI Helm repositories, Kubernetes secrets of type `docker-registry` are also supported.

Reviewer: ```suggestion
For OCI Helm repositories, Kubernetes secrets of type [kubernetes.io/dockerconfigjson](https://kubernetes.io/docs/concepts/configuration/secret/#secret-types) are also supported.
```

> It is possible to create one such secret with the following fields:

Reviewer: ```suggestion
It is possible to create one such secret with `kubectl create secret docker-registry` or using the Flux CLI:
```
